chaco/tools/regression_lasso.py

```diff
@@ -60,8 +60,8 @@ class RegressionOverlay(LassoOverlay):
         ),
     )
 
-    def _draw_component(self, gc, view_bounds=None, mode="normal"):
-        super()._draw_component(gc, view_bounds, mode)
+    def overlay(self, other_component, gc, view_bounds=None, mode="normal"):
```

Comment:
I tested this by running `examples/demo/basic/regression.py` and everything seems to work as expected